app/controllers/training/pages_controller.rb

```diff
@@ -100,7 +110,7 @@ def track_module_start?
 
   # @return [Boolean]
   def track_confidence_check_complete?
-    helpers.cms_module_progress(module_item.parent).completed? && untracked?('confidence_check_complete', training_module_id: mod_name)
+    module_item.thankyou? && untracked?('confidence_check_complete', training_module_id: mod_name)
```

Comment:
I'd like us to copy over all the tracking changes made in the YAML event realignment work to avoid any gaps in the migration. I'm not sure if there are any but I just spotted this doesn't use:

```ruby
  # @return [Boolean]
  def confidence_complete_untracked?
    untracked?('confidence_check_complete', training_module_id: training_module_name)
  end
```